tests/impl.cpp

```diff
@@ -1857,6 +1859,22 @@ bool test_mm_cvtps_epi32(const float _a[4])
     return validateInt32(ret, trun[0], trun[1], trun[2], trun[3]);
 }
 
+bool test_mm_testz_si128(const int32_t *_a, const int32_t *_b)
+{
+    __m128i a = _mm_load_si128((const __m128i *) _a);
+    __m128i b = _mm_load_si128((const __m128i *) _b);
+    int testz = 0;
+    int i;
+    for (i = 0; i < 2; i++) {
+        if ((((SIMDVec *) &a)->m128_u64[i] & ((SIMDVec *) &b)->m128_u64[i]) ==
+            0) {
+            testz = 1;
+            break;
```

Comment:
I think it should be
```c
int testz = 1;
...
for
  if(... != 0)
    testz = 0;
    break;
```

Reply:
Even we have 10K randomized input vectors for the test case, but it is still hard to diagnose the logic errors though.